Dockerfile

```diff
@@ -1,22 +1,41 @@
 # Base will install runtime dependencies and configure generics
-FROM node:12.16
+FROM node:12-slim as base
+LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>, Nico Stapelbroek <discord-soundbot@nstapelbroek.com>"
+
+RUN apt-get update -q && apt-get install -qy wget
+RUN wget -qO /tini https://github.com/krallin/tini/releases/download/v0.18.0/tini-$(dpkg --print-architecture) && \
+    chmod +x /tini && mkdir /app && chown -R node:node /app
 
-LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>"
 WORKDIR /app
 
-# Copy some files
-COPY package.json yarn.lock ./
-COPY . /app
 
-# Install ffmpeg and other deps
-RUN apt-get update
-RUN apt-get install --yes build-essential git ffmpeg make python > /dev/null
-RUN yarn install --silent
+# Builder will install system dependencies
+FROM base as builder
+RUN apt-get update -q && apt-get install -qy python make g++ tar xz-utils
+RUN wget -qO /tmp/ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-$(dpkg --print-architecture)-static.tar.xz && \
+    tar -x -C /usr/local/bin --strip-components 1 -f /tmp/ffmpeg.tar.xz --wildcards '*/ffmpeg' && rm /tmp/ffmpeg.tar.xz
```

Comment:
Can we get ffmpeg with apt-get as well, or is there a reason we need to do this here? Is it because ts also compile for each of the platforms?

Reply:
Ah, I now rediscovered why I did this:

1. It allows me to have the latest and greatest ffmpeg version
1. installing using apt blows up the image size, as we only need it to encode audio I think some of the installed dependencies of ffmpeg are not used. 
![Screenshot 2020-05-11 at 08 18 28](https://user-images.githubusercontent.com/3368018/81530082-69d60a80-9360-11ea-83df-785b7a0103c4.png)

Reply:
The size is definitely an argument!